WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -150,3 +150,60 @@ $venue-map-wrapper-height: 400px;
 .big-column {
   width: 100%;
 }
+
+.competition-select.form-inline {
+  background-color: #fff;
+  width: 100%;
+  padding-bottom: 10px;
+
+  input,
+  select {
+    font-size: 12px;
+    font-weight: normal;
+    display: block;
+  }
+
+  button {
+    width: 100px;
+  }
+
+  .btn-group {
+    display: block;
+  }
+}
+
+.same-line {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+.competition-list {
+  li:nth-child(odd) {
+    background-color: $table-bg-accent;
+  }
+
+  .date {
+    float: right;
+    font-weight: bold;
+  }
+}
+
+.competition-info {
+  p {
+    margin: 0;
+  }
+
+  // For the on-going competitions, we use the full width of the parent, so it doesn't
+  // make sense to put the <p> on top of each other. Not sure how this will look on mobile.
```

Comment:
Answer: bad. See

![image](https://cloud.githubusercontent.com/assets/277474/13725428/2aa123fe-e856-11e5-8032-56dcfb88c78e.png)

This custom styling for `.in-progress` should only kick in above a certain screen size (md? whenever the lists go back to side by side). You can accomplish this very easily with a `@media` query.